microk8s-resources/actions/common/utils.sh

```diff
@@ -632,25 +632,40 @@ create_user_kubeconfigs() {
   create_kubeconfig_x509 "kubelet.config" "system:node:${hostname}" ${SNAP_DATA}/certs/kubelet.crt ${SNAP_DATA}/certs/kubelet.key ${SNAP_DATA}/certs/ca.crt
 }
 
-create_worker_kubeconfigs() {
+create_worker_kubeconfigs_local_apiserver() {
   hostname=$(hostname | tr '[:upper:]' '[:lower:]')
   create_kubeconfig_x509 "proxy.config" "system:kube-proxy" ${SNAP_DATA}/certs/proxy.crt ${SNAP_DATA}/certs/proxy.key ${SNAP_DATA}/certs/ca.remote.crt
   create_kubeconfig_x509 "kubelet.config" "system:node:${hostname}" ${SNAP_DATA}/certs/kubelet.crt ${SNAP_DATA}/certs/kubelet.key ${SNAP_DATA}/certs/ca.remote.crt
 }
 
+create_worker_kubeconfigs_remote_apiserver() {
+  Create worker configs for an API server that is provided
+  # $1: API server IP
+  # $2: API server port
+  
+  hostname=$(hostname | tr '[:upper:]' '[:lower:]')
+  create_kubeconfig_x509 "proxy.config" "system:kube-proxy" ${SNAP_DATA}/certs/proxy.crt ${SNAP_DATA}/certs/proxy.key ${SNAP_DATA}/certs/ca.remote.crt $1 $2
+  create_kubeconfig_x509 "kubelet.config" "system:node:${hostname}" ${SNAP_DATA}/certs/kubelet.crt ${SNAP_DATA}/certs/kubelet.key ${SNAP_DATA}/certs/ca.remote.crt $1 $2
+}
+
```

Comment:
this can also be updated to accept optional arguments, right? so we would not need the separate `create_worker_kubeconfigs_local_apiserver`